docs/contributors/check-accessibility.md

```diff
@@ -0,0 +1,179 @@
+
+Note that some Orca features don’t work with certain combinations of Ubuntu releases and application toolkits. Read [Improve screen reader usability](https://canonical-ubuntu-desktop-documentation.readthedocs-hosted.com/en/latest/how-to/accessibility/improve-screen-reader-usability/).
+
+You need to get familiar with keyboard shortcuts to use the screen reader. Modifier keys are used in the shortcuts:
+
+{kbd}`Super`
+: Generally mapped to the {kbd}`Windows` key on Windows computers, and the {kbd}`Command` key on Macs.
+
+{kbd}`Orca`
+: By default, the Orca key is {kbd}`CapsLock` for the Laptop layout, and the {kbd}`Insert` key for the Desktop layout. You can [set the layout](https://canonical-ubuntu-desktop-documentation.readthedocs-hosted.com/en/latest/tutorial/get-started-with-the-screen-reader/#determine-your-keyboard-layout) in the Orca settings, typing {command}`orca -s` in the terminal.
+    
+| Action | Laptop layout | Desktop layout |
+|--------|-----------------------------------|------------------------------------|
+| Enable screen reader | {kbd}`Super+Alt+S` | {kbd}`Super+Alt+S` |
+| Quit screen reader | {kbd}`Super+Alt+S` | {kbd}`Super+Alt+S` |
+| Learn mode | {kbd}`Orca+H` (exit with {kbd}`Esc`) | {kbd}`Insert+H` (exit with {kbd}`Esc`) |
+| Orca preferences | {kbd}`CapsLock+Space` | {kbd}`Insert+Space` |
+| Toggle flat review | {kbd}`CapsLock+P` | {kbd}`Insert+-` on the numeric keypad |
+| Say all (in flat review) | {kbd}`CapsLock` and double-press {kbd}`;` | Double-press {kbd}`+` on the numeric keypad |
+| Read current line | {kbd}`CapsLock+I` | {kbd}`8` on the numeric keypad |
+| Read next line | {kbd}`CapsLock+O` | {kbd}`9` on the numeric keypad |
+| Read previous line | {kbd}`CapsLock+U` | {kbd}`7` on the numeric keypad |
+
+You can find further guidance in the [screen reader documentation](https://canonical-ubuntu-desktop-documentation.readthedocs-hosted.com/en/latest/how-to/accessibility/read-screen-aloud/).
+
+### Screen reading checklist
+
+- All interactive elements (buttons, fields, links…) have a label and a role that is read out loud by the screen reader
+    * Roles explain the function of the component (“button”, “text”, “link”, “combobox”). Check [GTK roles](https://docs.gtk.org/gtk4/enum.AccessibleRole.html).
+- There are no two or more elements with the same label in the same page
+    * For example, if there’s a single share button, it can be labeled “Share”. But if there’s more than one “Show” button, you should specify what it refers to: “Show description”, “Show gallery”.
+- All labels are short, descriptive and meaningful
+    * Take context into account: if you are on an app page, the share button should be labeled “Share”, not “Share this app” or “Share [app name]”: it’s generally redundant with the page title/headings.
+- All pages have a meaningful title or main heading
+- All sections within a page have a meaningful heading
```

Comment:
This implies that every section has to have a heading. I think something like "All section headings within a page are meaningful" would be closer to the intended meaning.

Reply:
The original meaning was precisely to encourage that every section had a heading. But I think that might be going too far, so changing it as you suggested.

Reply:
LGTM :)